README.md

```diff
@@ -169,11 +169,12 @@ Examples
 --------
 
 - audio/video ([demo](https://collab-project.github.io/videojs-record/examples/audio-video.html) / [source](https://github.com/collab-project/videojs-record/blob/master/examples/audio-video.html))
+- audio/screen ([demo](https://collab-project.github.io/videojs-record/examples/audio-screen.html) / [source](https://github.com/collab-project/videojs-record/blob/master/examples/audio-screen.html))
```

Comment:
this does not need to be so high up the list, it can be above or below the screen-only demo imo.

Reply:
Fixed in rebase 0200355